src/platform/test/functional/apps/dashboard/esql_controls/create_from_dropdown.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
```

Comment:
Good to have more functional tests here. FTR results LGTM. 25 runs might not be enough to capture any flakiness, but I'm okay to merge this - especially since you've been good at fixing up these tests if they become flaky.

Reply:
If it is flaky I will def look at it Devon, thanx for your review 🙇‍♀️